CefSharp/IRequest.cs

```diff
@@ -91,6 +92,12 @@ public interface IRequest : IDisposable
         bool IsReadOnly { get; }
 
         /// <summary>
+        /// Set the flags used in combination with CefURLRequest
+        /// </summary>
+        /// <param name="flags">Request Flags</param>
+        void SetFlags(UrlRequestFlags flags);
```

Comment:
See line `28` in this file?  I think it was someone's earlier intention that the flags be implemented via a `Flags` property with `get`/`set` accessors.

What do you think about implementing the getter and changing the interface to be a property instead of methods?

Reply:
I have tried implemented  getFlags, but it always return 0.

Reply:
OK fair enough.  Could you please post a comment when you are able to change the formatting and comments?

Reply:
Yes. I will do it 28.10.2017

Reply:
why is it  fail? :)